test/versioned/winston/helpers.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const helpers = module.exports
```

Comment:
Nitpicky thing, but if we do `module.exports.someFunction = someLambda` as below, when tests fail their stack traces will show anonymous functions instead of the function names you gave here. Is that correct? As a minor thing, I would like to see the function names.

Reply:
I'm not sure i understand.  I'm just assigning module.exports to helper so every function below is a named function on the export

Reply:
oh but i can make the helper functions named functions and not just arrow functions, n/m I'll fix this